### PR TITLE
Update the MIT.md file and some files

### PR DESCRIPTION
**Pull request by Alabi12**
## SUMMARY
The following were files worked on;
- add the redux files
- add functionality to the add and remove button
- connect the reducers to the store
- add the provider to the root container
- update the MIT.md file

### CODE REVIEW

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @Alabi12,

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️
- Project functionalities ✔️

## Required Changes ♻️
_Check the comments under the review._
- [ ] I would advice you to maintain naming style. I can see that the footer jsx is declared with small letter, that will not make the code run as React would attempt to read it as HTML tag whereas it is jsx, also, I can see that line 4 of the app component, the bookList. You used capital B for the bookList whereas it the name of the file is with small b.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 1]
```diff
@@ -2,9 +2,9 @@ import './App.css';
 import React from 'react';
 import { Routes, Route } from 'react-router-dom';
 import BookList from './components/BookList';
-import NavBar from './components/navBar';
+import NavBar from './components/NavBar';
 import Categories from './components/categories';
-import Footer from './components/footer';
+import Footer from './components/Footer';
```
Comment (BenMukebo) [inline src/App.js]: - [ ] As requested by the previous reviewer, Kindly make sure to rename your jsx files name,  instead use capitalize eg: BookList.js, Footer.js, ... 😉 ✍🏻

**Review by BenMukebo — CHANGES_REQUESTED** [review]
Hi @Alabi12, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight :clap: 
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please Tag me on (@BenMukebo) when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alabi12** [comment]
@BenMukebo 

Thanks for the review. Am highly impressed, however there are no comments to your request.

Kindly state the review the comments to guide with the correction .

File: src/App.js [hunk 2]
```diff
@@ -2,9 +2,9 @@ import './App.css';
 import React from 'react';
 import { Routes, Route } from 'react-router-dom';
 import BookList from './components/BookList';
-import NavBar from './components/navBar';
-import Categories from './components/categories';
-import Footer from './components/footer';
+import NavBar from './components/NavBar';
+import Categories from './components/Categories';
+import Footer from './components/Footer';
```
Comment (christianbiring1) [inline src/App.js:7]: - [ ] As you have changed the names of the modules here, Kindly you may consider also changing the files' names.

image

**Review by christianbiring1 — APPROVED** [review]
Hi @Alabi12 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
Iapproveokay

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Gitflow is followed. heavy_check_mark
- [ ] Professional README file. heavy_check_mark

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**